src/redux/actions/yachtActions.js

```diff
@@ -0,0 +1,39 @@
+import axios from 'axios';
+
+export const FETCH_YACHTS_SUCCESS = 'FETCH_YACHTS_SUCCESS';
+export const FETCH_SINGLE_YACHT_SUCCESS = 'FETCH_SINGLE_YACHT_SUCCESS';
+
+const baseURL = 'http://localhost:3001';
+
+const fetchYachtsSuccess = (yachts) => ({
+  type: FETCH_YACHTS_SUCCESS,
+  payload: yachts,
+});
+
+const fetchYachts = () => (dispatch) => {
+  const header = {
+    'Content-Type': 'application/json',
+    Authorization: localStorage.getItem('token'),
+  };
+  axios.get(`${baseURL}/v1/yachts`, header).then((response) => {
+    const yachts = response.data;
+    dispatch(fetchYachtsSuccess(yachts));
+  });
+};
```

Comment:
When dispatching `fetchYachts `function, I am receiving and 401 status error. This could be because of missing headers.
Try by changing the function to something like this:
```suggestion
const fetchYachts = () => (dispatch) => {
  const request = axios.create({
    headers: {
      'Content-Type': 'application/json',
      Authorization: localStorage.getItem('token'),
    },
  });

  request.get(`${baseURL}/v1/yachts`).then((response) => {
    const yachts = response.data;
    dispatch(fetchYachtsSuccess(yachts));
  });
};
```